src/components/shared/DetailsWidgetUser/DetailsWidgetUser.tsx

```diff
@@ -1,33 +1,37 @@
 import React from 'react';
 
-import HookedUserAvatar from '~users/HookedUserAvatar';
 import MaskedAddress from '~shared/MaskedAddress';
 import InvisibleCopyableAddress from '~shared/InvisibleCopyableAddress';
-import { useUser, Colony } from '~data/index';
-import { Address } from '~types';
+import { Address, Colony } from '~types';
+import UserAvatar from '~shared/UserAvatar';
+import { useGetCurrentUserQuery } from '~gql';
 
 import styles from './DetailsWidgetUser.css';
 
 const displayName = 'DetailsWidgetUser';
 
 interface Props {
-  colony?: Colony;
+  colony: Colony;
   walletAddress: Address;
 }
 
 const DetailsWidgetUser = ({ colony, walletAddress }: Props) => {
-  const UserAvatar = HookedUserAvatar({ fetchUser: false });
-  const userProfile = useUser(walletAddress);
-  const userDisplayName = userProfile?.profile?.displayName;
-  const username = userProfile?.profile?.username;
+  const { data } = useGetCurrentUserQuery({
```

Comment:
We should probably rename this query (on `master`, to avoid too many conflicts) as it doesn't get the current user as the name suggests but rather a user with a given address